client/src/components/Forums/ThreadDetails/ThreadDetails.tsx

```diff
@@ -96,6 +99,9 @@ const ThreadDetail = ({ forumId, threadId }: ThreadDetailProps) => {
           </div>
         ))}
       </div>
+      <div className="mt-4">
+        <strong>Total Posts:</strong> {totalPosts}
```

Comment:
From a UX perspective this feels a little weird seeing # of replies in ThreadDisplay and then # posts adds the original post to the count in this component. My vote would be either stick with # replies here or not have the count at all in this component

Reply:
I agree. I think just having the totals show on the ThreadsDisplay page is sufficient. Once the user navigates inside the thread they are already looking at all the posts.

Reply:
Pushed an update to remove it